app/presenters/ht_registration_presenter.rb

```diff
@@ -19,14 +19,18 @@ class HTRegistrationPresenter < ApplicationPresenter
   READ_ONLY_FIELDS = %i[sent received finished ip_address env].freeze
   JIRA_BASE_URL = URI.join(Otis.config.jira.site, "/jira/", "browse/").to_s.freeze
   FIELD_SIZE = 45
+  VALID_AFFILIATIONS = %r{^(faculty|staff|member|employee)}
+  AUTH_EXCEPTIONS = ["nypl"]
```

Comment:
Consider moving these to configuration

Reply:
Getting the affiliations regex into config requires YAML magic or `eval` and I'm not fired up about either of these. But the exceptions list seems to fit nicely.

Reply:
:+1: The valid affiliations seems less likely to change; if we find that we need to update them with any frequency we can see what we can do.